src/main/java/core/basesyntax/utils/UserValidator.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.utils;
+
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationException;
+
+public class UserValidator {
+    private static final int MIN_AGE = 18;
+    private static final int MIN_LOGIN_LENGTH = 6;
+    private static final int MIN_PASSWORD_LENGTH = 6;
+
+    public static void checkAge(User user) {
```

Comment:
1. Do not use static methods
2. Create one p`ublic void  validate()` method and call **private** methods checkAge/checkLogin/checkPassword inside it